data/emea-on-rails/emea-on-rails-2021/videos.yml

```
@@ -0,0 +1,143 @@
+---
+- title: "Democratizing the Fight Against Ruby Memory Bloat"
+  raw_title: Democratizing the Fight Against Ruby Memory Bloat - Hongli Lai
+  speakers:
+    - Hongli Lai
+  event_name: EMEA on Rails 2021
+  published_at: "2021-10-19"
+  description: |-
+    Ruby apps can use a lot of memory, but not for the reasons you think. I've discovered that as much as 70% of the memory usage is not caused by Ruby, but by the system's memory allocator! The good news is that there are technically simple solutions. So why isn't everybody using them? That's the bad news: they're cumbersome to deploy. We can say that the solutions are not "democratized".
+
+    I'm on a mission to allow everyone to easily get rid of Ruby memory bloat. In this talk I'll explain where Ruby memory bloat comes from, what the solutions are, and how I'm working on democratizing the solutions in the form of Fullstaq Ruby. I will also talk about my efforts to make Fullstaq Ruby a sustainable open source project.
+
+    This talk was delivered at EMEA on Rails, a virtual mega-meetup which took place on June 9, 2021.
+  video_provider: :youtube
```

Looks like this is a bug in https://github.com/adrienpoly/rubyvideo/blame/512527816cda529b09302f88140e9b3882ad823a/app/models/youtube/video_metadata.rb#L27 and we should update it to `"youtube"`.

For this, would you mind updating all the `video_provider` to just `youtube`:

```suggestion
  video_provider: youtube
```

This actually might also be the reason why the tests are failing.